style-guide/personal-pronoun.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
```

Comment:
If he/him or she/her, the message to the user should recommend use of "they" (as per the style guide) if a personal pronoun is absolutely needed.